### PR TITLE
feat: introduce isStatusUpdate flag in sendMessage APIs

### PR DESCRIPTION
**Pull request by PascalPrecht**
When sending a profile status update, the message has to be sent to
a specific channel that has the id `@PUBKEY`.

This commit introduces a flag that controls whether the message is
sent to the currently active channel, or tot he profile status channel.

The same is done for the `sendImage` API.

### CODE REVIEW

File: src/status/chat.nim [hunk 1]
```diff
@@ -125,8 +125,8 @@ proc updateContacts*(self: ChatModel, contacts: seq[Profile]) =
     self.contacts[c.id] = c
   self.events.emit("chatUpdate", ChatUpdateArgs(contacts: contacts))
 
-proc init*(self: ChatModel) =
-  let chatList = status_chat.loadChats()
+proc init*(self: ChatModel, pubKey: string) =
+  var chatList = status_chat.loadChats()
```
Comment (jrainville) [inline src/status/chat.nim:129]: Are those changes needed?

**Review by jrainville — COMMENTED** [review]
Looks good. I just wonder if the changes in status/chat.nim are needed

File: src/status/chat.nim (hunk 1 above)
Comment (PascalPrecht) [reply src/status/chat.nim:129]: Ah, good catch. Both changes are needed in #1551 
Could've moved them there. Do you want me to?

Comment (jrainville) [reply src/status/chat.nim:129]: Nah it's k

**Review by jrainville — APPROVED** [review]

**Pull request merged**